source/modules/MEI.gestural.xml

```diff
@@ -181,6 +181,13 @@
       </attDef>
     </attList>
   </classSpec>
+  <classSpec ident="att.damp.ges" module="MEI.gestural" type="atts">
+    <desc xml:lang="en">Gestural domain attributes.</desc>
+    <classes>
+      <memberOf key="att.duration.ges"/>
+      <memberOf key="att.timestamp2.ges"/>
+    </classes>
```

Comment:
This should be `empty`, as damp is a single event without extend.

Reply:
OK

Reply:
Still there.

Reply:
@bwbohl it should be like this:

```suggestion
    <classes>
      <empty/>
    </classes>
```